dotCMS/src/main/java/com/dotmarketing/tag/business/TagFactoryImpl.java

```diff
@@ -89,56 +91,50 @@ public TagFactoryImpl () {
     }
 
     @Override
-    public List<Tag> getTagsByHost ( String hostId ) throws DotDataException {
+    public List<Tag> getTagsByHost (final String siteId) throws DotDataException {
 
-        List<Tag> tags = tagCache.getByHost(hostId);
+        List<Tag> tags = tagCache.getByHost(siteId);
         if ( tags == null ) {
-
-            hostId = escapeSingleQuote(hostId);
```

Comment:
is it possible to add an IT to test this case?

Reply:
We already have tests but I added more tests and fixed a problem in code.